examples/examples.go

```diff
@@ -141,6 +141,12 @@ func RandomAccount(flowClient *client.Client) (flow.Address, *flow.AccountKey, c
 	return account.Address, account.Keys[0], signer
 }
 
+func GetReferenceBlockId(flowClient *client.Client) flow.Identifier {
+	blk, err := flowClient.GetLatestBlock(context.Background(), false)
```

Comment:
It's probably better to set `isSealed=true` here:
```suggestion
	block, err := flowClient.GetLatestBlock(context.Background(), true)
```